packages/fxa-auth-server/lib/senders/emails/templates/subscriptionSubsequentInvoiceDiscount/en.ftl

```diff
@@ -27,3 +27,10 @@ subscriptionSubsequentInvoiceDiscount-content-subtotal = Subtotal: { $invoiceSub
 # Variables:
 #  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
 subscriptionSubsequentInvoiceDiscount-content-discount = Discount: -{ $invoiceDiscountAmount }
+# Variables
+#  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
+subscriptionSubsequentInvoiceDiscount-content-discount-one-time = One Time Discount: -{ $invoiceDiscountAmount }
+# Variables
+#  $invoiceDiscountAmount (String) - The amount of the discount of the subscription invoice, including currency, e.g. $2.00
+#  $discountDuration - The duration of the discount in number of months, e.g. 3 months
+subscriptionSubsequentInvoiceDiscount-content-discount-repeating = { $discountDuration } Month Discount: -{ $invoiceDiscountAmount }
```

Comment:
I think this is a question for @annasob I don't think we have a way to tell what month they are into their discount. I'll say, personally, if I did a 3 month coupon to something and the second invoice said 3 month coupon I wouldn't find it that confusing but I think this is the copy that was approved by legal?

As i mentioned this isn't specified in the ticket and we dont really have a way to specifiy what month they are in? And even if so, I'm not sure adding that level of specificity is necessary? (question marks to indicate uncertainty)

Reply:
Cool, @annasob confirmed that we can leave it as is.

Just as an FYI, the discount object does have start and end dates available, so it would be possible to calculate how many months are left. Just in case we need to do something like that in future.